cli/tauri.js/src/helpers/rust-cli.ts

```diff
@@ -42,6 +46,14 @@ export function runOnRustCli(
       process.cwd(),
       onClose
     )
+  } else if (process.env.NODE_ENV === 'production') {
+    await downloadCli()
+    pid = spawn(
```

Comment:
If it aint broke, don't fix it? I don't see much of a reason to switch to execa since it all seems to work fine now, and it would probably best to have it as a separate PR as it isn't really related to this PR.

Reply:
👍